Updating WebGrab+Plus to 3.1.4

Laibalion opened the pull request:
## Description:
Just updating the used WebGrab+Plus version from 3.1.1 beta to 3.1.4 beta

aptalca commented:
I'll let the maintainer decide on the rest of it, but you shouldn't modify an old entry in the changelog. You should add a new line instead. Otherwise it would only confuse people.

Laibalion commented:
> I'll let the maintainer decide on the rest of it, but you shouldn't modify an old entry in the changelog. You should add a new line instead. Otherwise it would only confuse people.

You are right, my mistake.
Updated the PR to not modify the readme file itself and added entry to readme yml instead of replacing it
